alws/routers/repositories.py

```diff
@@ -27,3 +28,13 @@ async def get_repository(repository_id: int, db: database.Session = Depends(get_
     if result:
         return result[0]
     return None
+
+
+@router.post('/exports/', response_model=typing.List[str])
```

Comment:
I think we need a full CRUD at least for exporters, to be able to see what we have and maybe modify it

Reply:
Hm. Not sure I got the idea

Reply:
Done